lib/badge_queries.rb

```diff
@@ -160,11 +160,15 @@ def self.invite_badge(count, trust_level)
         FROM invites i
         JOIN invited_users iu ON iu.invite_id = i.id
         JOIN users u2 ON u2.id = iu.user_id
-        WHERE i.deleted_at IS NULL AND u2.active AND u2.trust_level >= #{trust_level.to_i} AND u2.silenced_till IS NULL
+        WHERE i.deleted_at IS NULL
+        AND i.invited_by_id <> u2.id
+        AND u2.active
+        AND u2.trust_level >= 0
```

Comment:
I'm not sure they meant to actually replace the trust level from the method call with 0. 🤔 I believe what they pasted was the query after interpolation with some example values?

Reply:
@Drenmi Thanks for pointing that out. I fixed that.